h5p-bildetema/src/components/CollectionsController/CollectionController.module.scss

```diff
@@ -0,0 +1,61 @@
+@use "common/abstracts" as *;
+
+.CollectionController {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.mainSize {
+  margin: $main-content-margin;
+  width: $main-content-width;
+  max-width: $main-content-max-width;
+  margin-bottom: 2rem;
+}
+.menuWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 2rem 0rem;
+}
+
+// remove
```

Comment:
Why do we have this here? Should it be removed? Maybe add more info if we should keep the comment

Reply:
Yes, i forgot to remove it 😅